src/ingestion/process-event.ts

```diff
@@ -0,0 +1,66 @@
+import { KafkaConsumer, Producer, ProducerStream } from '@posthog/node-rdkafka'
+import { DateTime } from 'luxon'
+import { PluginsServer, EventData, Properties, Queue } from 'types'
+import { UUIDT } from '../utils'
+import { KAFKA_EVENTS, KAFKA_SESSION_RECORDING_EVENTS } from './topics'
+import { Pool } from 'pg'
+
+export class EventsProcessor implements Queue {
```

Comment:
This looks like an empty class. I guess you have plans for it in the future, but then I would separate `process_event_ee` and anything that has to do with kafka into two separate lumps of code.

Reply:
👍 Can clean that up.